scripts/qa_crowbarsetup.sh

```diff
@@ -6044,6 +6064,10 @@ function onadmin_batch
     if grep -q "barclamp: tempest" ${scenario}; then
         safely crowbar batch --include tempest --timeout 3600 build ${scenario}
     fi
+
+    # enable OpenID Connect by default
+    update_one_proposal keystone default
```

Comment:
so this part I find rather dangerous, it changes the behavior of crowbar batch to also modify attributes over what is specified in crowbar batch?

why don't you instead simply update the template in  ./scripts/scenarios/cloud9/cloud9-2nodes-default.yml instead? that should do what you want..

Reply:
Thanks for the suggestion Dirk! Yeah, originally I was thinking enable it for all scenarios as it doesn't carry that much overhead. But I am fine with enabling it for the 2nodes-default only. Let me make the change.